pkg/apiclient/application/forwarder_overwrite_test.go

```diff
@@ -0,0 +1,68 @@
+package application
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+)
+
+func toMap(obj interface{}) (map[string]interface{}, error) {
```

Comment:
Can `toMap` method be moved to utils? Often we need to convert to interface to map